docker-compose-dev.yml

```diff
@@ -20,5 +20,8 @@ services:
       - ./:/umdio
     ports:
       - 3000:3000
+      - 1234:1234
```

Comment:
What are these ports for?

Reply:
artifacts from trying `ruby_debug_ide`. removed.